### PR TITLE
View

### PR DESCRIPTION
**Pull request by malikhaiderkhan**
In this part of project I implemented:

- The design from the sneak peek wireframes.
- Use methods that I have created in the Blog App - processing data in models.

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,29 @@
...
+</div>
+
+<div class="justify-content-center row align-items-center m-4">
+<div class="border p-2 col-lg-8 col-11">
+  <h5>Bio</h5>
+  <p><%= @user.bio %></p>
+</div>
+</div>
+
+<% if @user.posts.any? %>
```
Comment (Lucas-Masaba) [inline app/views/users/show.html.erb:13]: - [ ] Please use the `three_recent_posts` method to render the 3 most recent posts as mentioned in the previous requirements.

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1 +1,26 @@
...
+
+<% @user.posts.each_with_index do |post, index| %>
+  <a href="/users/<%= @user.id %>/posts/<%= post.id %>" class="text-decoration-none">
+    <%= render "partials/post", post: post, post_index: index + 1 %>
+    </a>
+    
+    <div class="justify-content-center align-items-center row mt-1">
+      <div class="border p-2 col-lg-8 col-md-11 col-11 ">
+        <% if post.comments.any? %>
+          <% post.comments.first(5).each do |comment| %>
```
Comment (Lucas-Masaba) [inline app/views/posts/index.html.erb]: - [ ] Please use the `five_recent_comments` method you created in the Post model to render the 5 most recent comments as mentioned in the previous requirements.

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @malikhaiderkhan  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Linter checks passing :heavy_check_mark: 
- Well-written commit messages :writing_hand: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @malikhaiderkhan,

✔️ Implementation of wireframe layouts across pages

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**